taxi/forms.py

```diff
@@ -0,0 +1,40 @@
+from django import forms
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+from taxi.models import Driver
+
+
+def check_license_number(number: str):
+    if not number[0:3].isalpha() or not number[0:3].isupper():
+        raise ValidationError("First 3 letters must be capitals")
+    elif not number[3:9].isdigit():
+        raise ValidationError("Last 5 letters must be digits")
+
+    return number
+
+
+class DriverCreationForm(UserCreationForm):
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = UserCreationForm.Meta.fields + ("license_number",)
```

Comment:
would be good to add first_name and last_name to the form as well

Reply:
I'll add.